docs/cmd/tkn_bundle_pull.md

```diff
@@ -0,0 +1,51 @@
+## tkn bundle pull
```

Comment:
Just need another rename in here for "tkn bundle list" and we should be good to go,

Reply:
Just when you think you have them all...

I went ahead and removed this doc since now its in `doc/cmd/tkn_bundle_list`